spec/state_machines/global_state_machine_spec.rb

```diff
@@ -0,0 +1,1387 @@
+
+require 'rails_helper'
+
+describe 'state machine' do
+
+  before(:all) do
+    DbHousekeeping.clean
+    @setup = StandardSetup.new
+  end
+
+  after(:all) { DbHousekeeping.clean }
+
+  describe :accept_approver_assignment do
+    it {
+      expect(1).to eq 1
```

Comment:
Checking that the fundamental laws of computing still hold? ;)

Reply:
yep